src/main/scala/org/graphframes/GraphFrame.scala

```diff
@@ -450,12 +419,15 @@ object GraphFrame extends Serializable {
   /** Column name for vertex IDs in [[GraphFrame.vertices]] */
   val ID: String = "id"
 
-  /** Column name for source vertex IDs in [[GraphFrame.edges]] */
+  /** Column name for source vertex IDs in [[GraphFrame.edges]] and [[GraphFrame.triplets]] */
```

Comment:
If I understand correctly the triplets, this is not quite true:
 - it is the name of the column that contains the source IDs in edges
 - but it is the name of the aggregate column that contains all the vertex attributes of the source edge in the case of the triplets (this aggregate contains among other data the vertex id)